packages/volto/news/5580.feature

```diff
@@ -0,0 +1 @@
+Expose robots.txt setting in Volto control panel, and render robots.txt based on REST API call. @robgietema
```

Comment:
Where is it exposed in the Volto control panel? I don't see it there. I expect it needs to be removed from https://github.com/plone/volto/blob/5c88b0b767bbca7edf36467e16faf827f288c02c/packages/volto/src/config/ControlPanels.js#L79

Reply:
Done, but related to the other comment: 

<img width="1137" alt="image" src="https://github.com/user-attachments/assets/c36a1c37-6ef3-4cdc-81cf-66255c87c7f5" />

The backend answer with the .gz, which we will override it afterwards in a shady way in code (btw, we are doing it right now)

Reply:
@sneridagh @ericof Let's change the sitemap to use sitemap-index.xml in the backend setting in https://github.com/plone/plone.volto/pull/183